client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/Configuration.java

```diff
@@ -204,6 +210,10 @@ public Features features() {
       return features;
    }
 
+   public Collection<SerializationContextInitializer> getContextInitializers() {
```

Comment:
As I was saying, List. I know the actual impl is using ArrayList but I'd like to see List at API level also